packages/lesswrong/components/sequences/LargeSequencesItem.tsx

```diff
@@ -194,11 +195,21 @@ export const LargeSequencesItem = ({sequence, showAuthor=false, classes}: {
         </div>
       </div>
       <div className={classes.right}>
-        {posts.map(post => <SequencesSmallPostLink 
-            key={sequence._id + post._id} 
-            post={post}
-            sequenceId={sequence._id}
-          />
+        {sequence.chapters?.flatMap(({posts, contents}) =>
+          <>
+            {showChapters && contents?.htmlHighlight && (
+              <ContentStyles contentType="postHighlight">
+                <ContentItemBody dangerouslySetInnerHTML={{__html: contents.htmlHighlight}} />
+              </ContentStyles>
+            )}
```

Comment:
Why are we showing the html highlight instead of the chapter title?

Reply:
@jpaddison3  The reason for this is that Jesse and Will have been using the chapter body instead of the title in order to be able to format the text:
<img width="395" alt="Screenshot 2022-07-27 at 10 34 24" src="https://user-images.githubusercontent.com/5075628/181214877-33bbf79c-9dc0-4373-910a-c1a1312f75bb.png">

I spoke to Will about this a couple of days ago and it seems like they're just having some general issues around splitting the content up in the way that they feel is best for the curriculum, and this seemed like the best compromise so they can make the subdivisions a bit clearer.

I might need to talk with Will again though because it looks like there's one chapter called "Some newer EA strategies for improving the wellbeing of humans living in poverty (note that these are just two examples of interventions that EAs have been working on - there are many more in the 'more to explore' sections.):", which is probably way to long to fit nicely.

This might need some tweaking in the future once the content's finalized, but perhaps something like `title ?? contents.htmlHighlight` would be a good compromise for now?

Reply:
> perhaps something like title ?? contents.htmlHighlight would be a good compromise for now

Seems good